app/views/shared/_notes.html.erb

```diff
@@ -3,13 +3,8 @@ chapter_note ||= false %>
 
 <% if section_note || chapter_note %>
   <article class="notes">
-
-    <% if section_note && chapter_note %>
-      <h2 class="heading-small">There are important section notes and chapter notes for this part of the tariff:</h2>
-    <% end %>
-
     <% if chapter_note %>
-      <div id="chapter-notes">
+      <div id="notes">
```

Comment:
think we can keep these and change the class above to an id and achieve the same thing, its best not to have two elements with the same id